docs/en/connector-v2/sink/Easysearch.md

```diff
@@ -0,0 +1,178 @@
+# INFINI Easysearch
+
+## Description
+
+A sink plugin which use send data to `INFINI Easysearch`.
+
+## Key features
+
+- [ ] [exactly-once](../../concept/connector-v2-features.md)
+- [x] [cdc](../../concept/connector-v2-features.md)
+
+:::tip
+
+Engine Supported
+
+* Supported all versions released by [INFINI Easysearch](https://www.infini.com/download/?product=easysearch).
+
+:::
+
```

Comment:
![Uploading image.png…]()

Data Mapping Lost. you can refer https://seatunnel.apache.org/docs/2.3.3/connector-v2/sink/Mysql

Reply:
updated